R-package/cran-comments.md

```diff
@@ -1,5 +1,120 @@
 # CRAN Submission History
 
+## v3.3.2 - Submission 1 - (TBD)
```

Comment:
#4930 couldn't be merged into `master` and was closed, so we need to update CRAN comments file here.